metadata_mapper/lambda_function.py

```diff
@@ -45,6 +45,25 @@ def parse_enrichment_url(enrichment_url):
     return enrichment_func, kwargs
 
 
+def run_enrichments(records, payload, enrichment_set):
+    collection = payload.get('collection', {})
+    for enrichment_url in collection.get(enrichment_set, []):
+        enrichment_func, kwargs = parse_enrichment_url(enrichment_url)
+        if not enrichment_func and settings.SKIP_UNDEFINED_ENRICHMENTS:
```

Comment:
Is it possible for enrichment_func to be None and for settings.SKIP_UNDEFINED_ENRICHMENTS to be False? Do we want to handle that case?

Reply:
In `parse_enrichment_url` I raise an exception if SKIP_UNDEFINED_ENRICHMENTS is False and enrichment_func is None.